decode/evm_rpc.go

```diff
@@ -89,6 +89,20 @@ func MethodHasBlockHashParam(method string) bool {
 	return includesBlockHashParam
 }
 
+var StaticMethods = []string{
```

Comment:
@evgeniy-scherbina any reason not to add all the known static methods in this pr?

Reply:
Discussed, final list:
```go
var StaticMethods = []string{
	"eth_chainId",
	"net_version",
}
```